vllm/config.py

```diff
@@ -50,6 +50,7 @@
     "Qwen2ForCausalLM",
     "Qwen2MoeForCausalLM",
     "QWenLMHeadModel",
+    "InternVLChatModel"
```

Comment:
We can add `"InternLM2ForCausalLM"` here as well, otherwise the PP for InternLM2 backbone won't be enabled.